### PR TITLE
feat: Implement _mm_movepi64_pi64 and _mm_movpi64_epi64

### PR DESCRIPTION
**Pull request by marktwtn**

### CODE REVIEW

File: sse2neon.h [hunk 1]
```diff
@@ -2134,6 +2134,28 @@ FORCE_INLINE int _mm_movemask_epi8(__m128i a)
...
+// Copy the 64-bit integer a to the lower element of dst, and zero the upper
+// element.
+//
+//   dst[63:0] := a[63:0]
+//   dst[127:64] := 0
+//
+// https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=_mm_movpi64_epi64
+FORCE_INLINE __m128i _mm_movpi64_epi64(__m64 a)
+{
+    return _mm_set_epi64x(0, (int64_t) vreinterpret_s64_m64(a));
```
Comment (jserv) [inline sse2neon.h:2156]: You can reimplement as following:
```c
vcombine_s64(vreinterpret_s64_m64(a), vdup_n_s64(0));
```

Comment (marktwtn) [reply sse2neon.h]: Done.

**Review by jserv — CHANGES_REQUESTED** [review]
Fix conflicts

**Comment by marktwtn** [comment]
The conflict has been eliminated.

**Pull request merged**